egui/src/widgets/button.rs

```diff
@@ -142,15 +165,28 @@ impl Widget for Button {
         }
         desired_size = desired_size.at_least(min_size);
 
+        if let Some(image) = image {
+            let image_padding = image.size() + 4.0 * button_padding;
+            desired_size.x += image_padding.x;
+            desired_size.y = image_padding.y;
```

Comment:
```suggestion
            desired_size.x += image.size().x + ui.spacing().icon_spacing;
            desired_size.y = desired_size.y.max(image.size().y + 2.0 * button_padding.y);
```

Reply:
why 4x `button_padding` for images?

Reply:
You use `icon_spacing` below, but not here - that's inconsistent.

Reply:
Thanks, I've updated per the suggested change. I believe I was trying to go based off of the padding of the button frame and then extend that, based on some of the ways the radio and other buttons that had text near them, but that didn't work as seen by the shifting too far to the left. The new calculation does look much better.

![image](https://user-images.githubusercontent.com/4337981/140825703-139c9199-1935-4f2c-930f-5a78361b7c87.png)

![image](https://user-images.githubusercontent.com/4337981/140825750-267bd665-5e9c-47a9-b5fa-76f93f9beed7.png)